.travis.yml

```
@@ -1,8 +1,18 @@
 language: java
 
-jdk:
-  - openjdk8
-  - openjdk9
-  - openjdk10
-  - openjdk11
-  - openjdk12
+before_script:
+- wget https://github.com/sormuras/bach/raw/master/install-jdk.sh
+
+matrix:
+  include:
+  - jdk: openjdk8
+  - env: JDK=9
+    before_install source ./install-jdk.sh --feature $JDK
+  - env: JDK=10
+    before_install source ./install-jdk.sh --feature $JDK
+  - env: JDK=11
+    before_install source ./install-jdk.sh --feature $JDK
+  - env: JDK=12
+    before_install source ./install-jdk.sh --feature $JDK
+  - env: JDK=adoptopenjdk11 + OpenJ9
```

does this want some quotes? like:
```suggestion
  - env: "JDK=adoptopenjdk11 + OpenJ9"
```